### PR TITLE
MCR-2439 read and change object state via rest api

### PR DESCRIPTION
**Pull request by yagee-de**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2439).

### CODE REVIEW

**Review by kkrebs — CHANGES_REQUESTED** [review]
We should discuss this. For details see comment in jira ticket.

**Comment by kkrebs** [comment]
there are conflicts, please rebase branch and resolve conflicts

File: mycore-restapi/src/main/java/org/mycore/restapi/MCRApiDraftFilter.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package org.mycore.restapi;
```
Comment (rsteph-de) [inline mycore-restapi/src/main/java/org/mycore/restapi/MCRApiDraftFilter.java:19]: License Header?

File: mycore-restapi/src/main/java/org/mycore/restapi/MCRApiDraftFilter.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import javax.ws.rs.container.ContainerRequestFilter;
+import javax.ws.rs.container.ResourceInfo;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+
+import org.apache.logging.log4j.LogManager;
+import org.mycore.common.config.MCRConfiguration2;
+import org.mycore.restapi.annotations.MCRApiDraft;
+
+@Priority(Priorities.AUTHORIZATION)
```
Comment (rsteph-de) [inline mycore-restapi/src/main/java/org/mycore/restapi/MCRApiDraftFilter.java:48]: JavaDoc?

File: mycore-restapi/src/main/java/org/mycore/restapi/annotations/MCRApiDraft.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package org.mycore.restapi.annotations;
```
Comment (rsteph-de) [inline mycore-restapi/src/main/java/org/mycore/restapi/annotations/MCRApiDraft.java:19]: License Header

**Review by rsteph-de — CHANGES_REQUESTED** [review]
License header and Javadocs on new files are missing!

**Review by rsteph-de — APPROVED** [review]
looks good for me now!

**Review by kkrebs — APPROVED** [review]

**Pull request merged**